tools/headerparser/third_party/test_headers/bluetooth-power.h

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Better provide a single small test file that's not part of any existing project, otherwise we'll end up in a license hell :)